psi_cardinality/c/BUILD

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright 2020 the authors listed in CONTRIBUTORS.md
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "psi_cardinality_client_c",
```

Comment:
The `_c` suffix isn't necessary since the C bindings are now in their own directory.

Reply:
I actually think we can also remove the `psi_cardinality_` prefix and just leave it at `client` and `server`. But let's save that part for another PR.